menderqa_pipeline_helpers.go

```diff
@@ -9,8 +9,21 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func getLatestIntegrationRelease(number int, conf *config) ([]string, error) {
```

Comment:
This function does not belong here: `menderqa_pipeline_helpers.go` are meant to be used only from `menderqa_pipeline.go`. Maybe you want to put it in `util.go`, or directly in `cherrypicks.go` (it is not used anywhere else).

Reply:
moved